docs/csharp/modern-events.md

```diff
@@ -108,9 +108,8 @@ task that can report the error by entering the faulted state. Because
 the method is async, the method can't simply throw the exception. (The
 calling method has continued execution because it is `async`.) The
 actual runtime behavior will be defined differently for different
-environments. It may terminate the thread, it may terminate the program,
-or it may leave the program in an undetermined state. None of those
-are good outcomes.
+environments. It may terminate the thread, the process that owns the thread,
```

Comment:
```suggestion
environments. It may terminate the thread or the process that owns the thread,
```

Reply:
Thank you for the suggestion, @gewarren . I'm not an expert with english, but it's sounds strange to me the phrase with two "or". 

"It may terminate the thread *or* the process that owns the thread, *or* leave the process in..."

Are you sure about this?

Reply:
If you leave the original comma and omit the "or" I added, then the sentence means the following:

It may:
-terminate the thread
-the process that owns the thread (this part doesn't make sense because the verb is missing)
-leave the program in an undetermined state

Or the following, which also doesn't make sense:

It may terminate:
-the thread
-the process that owns the thread
-leave the process in an indeterminate state (now this part doesn't make sense because it has two verbs, terminate and leave)